docs/get-started/cli-commands.mdx

```diff
@@ -1279,6 +1280,122 @@ Address of the gRPC API. Default: `127.0.0.1:9632`.
 
 Address of the gRPC API. Default: `127.0.0.1:9632`.
 
+<h3>txpool subscribe flags</h3>
+
+<h4><i>grpc-address</i></h4>
+
+<Tabs>
+  <TabItem value="syntax" label="Syntax" default>
+
+    txpool subscribe [--grpc-address GRPC_ADDRESS]
+
+  </TabItem>
+  <TabItem value="example" label="Example">
+
+    txpool subscribe --grpc-address 127.0.0.1:10003
```

Comment:
I've noticed all grpc-address in cli-commands `127.0.0.1:10003`. I'm not sure this address is right.

For example, in PoA like this
```md
For a new node to become a validator at least 51% of validators need to propose him.

Example of how to propose a new validator (`0x8B15464F8233F718c8605B16eBADA6fc09181fC2`) from the existing validator node on grpc address: 127.0.0.1:10000:


polygon-sdk ibft propose --grpc-address 127.0.0.1:10000 --addr 0x8B15464F8233F718c8605B16eBADA6fc09181fC2 --vote auth
```